packages/compass-logging/src/index.ts

```diff
@@ -0,0 +1,34 @@
+import { MongoLogWriter, mongoLogId } from 'mongodb-log-writer';
+import { Writable } from 'stream';
+
+function createLogger(component: string): {
+  log: ReturnType<MongoLogWriter['bindComponent']>;
+  mongoLogId: typeof mongoLogId;
+} {
+  // hadron-ipc only works inside of Electron, hence the conditional
+  let ipc: typeof import('hadron-ipc') | undefined;
+  try {
+    ipc = require('hadron-ipc');
+  } catch {
+    /* not electron */
```

Comment:
I feel like the change in https://github.com/mongodb-js/compass/pull/2428 is a bit out of place, it seems fairly unrelated to the rest of that PR … either way, this isn’t too hard to adjust in whichever PR lands second.

Reply:
If i recall it correctly it was necessary so tests in other plugins would not explode .. if you want i can do something about it in that PR after you merge this one

Reply:
I mean, this here is also only necessary so that the tests for this particular package work.

Let me do something about this, with the 2pm meeting being cancelled I think the time’s there :)

Reply:
Ok, the two PRs are aligned now :+1:

Reply:
Ok great, thanks!